addons/binding/org.openhab.binding.lutron/ESH-INF/thing/thing-types.xml

```diff
@@ -167,6 +167,154 @@
         </config-description>
     </thing-type>
 
+    <bridge-type id="prgbridge">
+        <label>Lutron GRX-PRG or GRX-CI-PRG Bridge</label>
+        <description>Ethernet access point to Lutron Grafik Eye 3x/4x Systems</description>
+
+        <channels>
+            <channel id="buttonpress" typeId="buttonpress"/>
```

Comment:
It's a bit more complicated actually.  First digit is the "address" that was selected (upper case for button press, lower case for button release), the second digit was "the result".  The first digit will be either a controller or keypad address.  The second digit (if a keypad) is entirely dependent upon the physical type of keypad and it's configuration.

While technically you are correct in that I can probably abstract this away into proper channels, I doubt anyone would really use those channels (you really only care about the new scene and it's settings - which you get through the other channels).  I didn't think it's worth the time/effort to do that and if someone really needed that, they can use this channel to parse what they need (probably are looking for a very specific button press anyway).